modules/core/client/views/footer.client.view.html

```diff
@@ -0,0 +1,10 @@
+<div class="container" ng-controller="FooterController as vm">
+  <div class="row">
+    <div class="col-md-12">
+      <hr/>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-md-4 col-md-offset-4">&copy; {{vm.year}} MEAN.js | All rights reserved</div>
```

Comment:
>How would you go about getting the server config to the client?

a very good question.

For instance:

1. Have settings object next to user object at [the layout](https://github.com/meanjs/mean/blob/master/modules/core/server/views/layout.server.view.html#L48).

2. Feed `$window.settings` object for Modules as a [Constant](https://github.com/johnpapa/angular-styleguide#style-y241) (_"Use constants for values that do not change and do not come from another service."_) or via Service.